homeassistant/components/webostv/media_player.py

```diff
@@ -450,6 +451,6 @@ async def async_button(self, button):
         await self._client.button(button)
 
     @cmd
-    async def async_command(self, command):
+    async def async_command(self, command, **kwargs):
         """Send a command."""
-        await self._client.request(command)
+        await self._client.request(command, payload=kwargs.get(ATTR_PAYLOAD))
```

Comment:
Does the client handle a payload of `None`?

Reply:
Yeap it does, it just doesnt send a payload when None is provided.

Reply:
Ok. Then we just need to adjust the test assertion.